### DISCUSSION TITLE
810

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#985] [Comments]
# 810

지난번 독서모임에서 The Goal에 대해 이야기 나누면서 우리 팀의 문화에 대해 소개 한 적이 있다.팀 내부에선 품앗이 문화라고 부르는데, 이 내용이 엔비디아의 조직 문화에 대한 글에서도 똑같이 소개하고 있어서 대단히 반가웠다.예전에 김영재 님이 페이스북에 써주신 "RnR 따지는 사람이 범인이다" 라는 이야기가 되게 기억에 오래 남아있다."잘되는 조직은 모두가 하나의 목표를 위해 니일내일 구분 없이 하는 것이 아닐까" 하는 생각을 했다.그래서 어떻게 하면 RnR을 따지지 않는 조직을 만들 수 있을까를 많이 생각했고, 인프랩 오고나서도 그런 조직을 만들고 싶었다.기본적인 직무 전문성을 보고 채용을 하기 때문에 RnR을 없앨 수도 없고 RnR을 없애는 것을 원하는 것도 아니다.기본적인 RnR은 당연히 존재..

https://jojoldu.tistory.com/810

**Comment by msbaek** [comment]
예전 모 커머스 회사는 기술력이 그렇게 뛰어나지는 않았지만 직원들의 근속년수가 높아서 손이 부족한 옆 부서 일을 도와줄 수 있어서 기여할 수 있다고 들었습니다. 매우 부러운 조직 구조라고 생각했고요.
엔비디아 용병도 그런 제도로 보이네요. 역시 잘 나가는 조직은 다 그들만의 이유가 있군요.

RnR을 따져서 FE는 다 했는데, BE는 다 했는데... 이런게 무슨 소용이 있겠어요.
회사/서비스의 입장에서 업무가 완결적으로 완료되어 성과에 기여할 때야 의미가 있겠죠 ㅎㅎ

그럴려면 내가 하는 업무(BE라면) 옆의 업무(FE, DevOps 등)들도 애지간히는 할 수 있어야 할 것 같아요.

근데 저 같은 경우는 매우 드물게 다루는 FE는 참 어렵게 느껴지기는 합니다. ㅎㅎ
